src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +36,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        Optional<User> maybeUser = Optional.ofNullable(user);
+        return maybeUser.filter(user1 -> user1.getAge() != null)
+                .filter(user1 -> user1.getAge() >= 30 && user1.getAge() <= 45)
+                .isPresent();
```

Comment:
아래와 같은 방법으로도 작성이 가능할 것 같아요. 😄 

```suggestion
        return Optional.ofNullable(user)
                .map(User::getAge)
                .filter(age -> age >= 30 && age <= 45)
                .isPresent();
```